tests/greenheart/test_hydrogen/test_electrolyzer_physics.py

```diff
@@ -0,0 +1,157 @@
+from pytest import approx
+import pandas as pd
+import greenheart.tools.eco.electrolysis as he_elec
+from hopp.utilities import load_yaml
+import os
+import unittest
+
+dirname = os.path.dirname(__file__)
+input_library_path = os.path.join(dirname, "input_files","plant")
+project_life_years = 30
+
+TOL = 1e-3
+
+class TestOffGridSimulation(unittest.TestCase):
+    def setUp(self) -> None:
+        return super().setUp()
+    
+    @classmethod
+    def setUpClass(self):
+        super(TestOffGridSimulation, self).setUpClass()
+        electrolyzer_size_mw = 960
+        grid_connected = False
+        cluster_rating_MW = 40
+
+        power_profile_filename = 'GS_offgrid_power_signal.csv'
+        offgrid_power_profile_filename = os.path.join(input_library_path,power_profile_filename)
+        offgrid_power_profile = pd.read_csv(offgrid_power_profile_filename)
+
+        greenheart_config_filename = os.path.join(input_library_path,"GS_greenheart_config.yaml")
+        greenheart_config = load_yaml(greenheart_config_filename)
+
+        greenheart_config['electrolyzer']['rating'] = electrolyzer_size_mw
+        greenheart_config['project_parameters']['grid_connection'] = grid_connected
+        greenheart_config['electrolyzer']['cluster_rating_MW'] = cluster_rating_MW
+
+        electrolyzer_physics_results = he_elec.run_electrolyzer_physics(offgrid_power_profile, project_life_years, greenheart_config, wind_resource = None, design_scenario='off-grid', show_plots=False, save_plots=False, verbose=False)
+        self.H2_Res = electrolyzer_physics_results["H2_Results"]["new_H2_Results"]
+        self.power_profile = electrolyzer_physics_results["electrical_generation_timeseries"]
+    
+    def test_AEP_input_power(self):
```

Comment:
Can you add a note about where the test values came from? Are these from the onshore Greensteel code?

Reply:
added a comment to the start of tests that explain that